course/grades.py

```diff
@@ -118,9 +119,6 @@ def view_participant_grades(pctx, participation_id=None):
             if not (opp.shown_in_grade_book
                     and opp.shown_in_participant_grade_book):
                 continue
-        else:
-            if not opp.shown_in_grade_book:
-                continue
```

Comment:
This is redundant as it had been done at this [line](https://github.com/inducer/relate/pull/466/files#diff-63d070b8acfd5aa88cf07944caf3f32cL104)

Reply:
OK